server/src/test/java/org/apache/iotdb/db/integration/IoTDBLastIT.java

```diff
@@ -103,6 +100,30 @@ public void tearDown() throws Exception {
     EnvironmentUtils.cleanEnv();
   }
 
+  @Test
+  public void lastWithEmptySeriesTest() throws Exception {
+    String[] retArray = new String[]{
+            "root.ln.wf02.temperature,null",
+        };
+
+    try (Connection connection =
+             DriverManager.getConnection("jdbc:iotdb://127.0.0.1:6667/", "root", "root");
+         Statement statement = connection.createStatement()) {
+
+      statement.execute(
+          "CREATE TIMESERIES root.ln.wf02.temperature WITH DATATYPE=DOUBLE, ENCODING=PLAIN");
+      statement.execute("select last temperature from root.ln.wf02");
```

Comment:
how about select two timeseries in a sql, while one time series has not data and another one has data.

Reply:
could add one more test for this